pdr_backend/cli/cli_arguments.py

```diff
@@ -112,6 +115,43 @@ def add_argument_LOOKBACK(self):
         )
 
 
+@enforce_types
+class NUM_Mixin:
+    def add_argument_NUM(self):
+        self.add_argument("NUM", type=int, help="1|2|..")
```

Comment:
I think you can leave the default help from "int" type. If you want this to be positive, you can also implement a custom type function or validate it when running the command.

Ref here: https://stackoverflow.com/a/14117511

Reply:
good work, done